pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -255,6 +259,10 @@ func (c *Config) DefaultEchoConfigs(t resource.Context) []echo.Config {
 		Ports:          ports.All(),
 		Subsets: []echo.SubsetConfig{{
 			Annotations: echo.NewAnnotations().Set(echo.SidecarInterceptionMode, "TPROXY"),
+			Labels: map[string]string{
+				label.SidecarInject.Name:     "false",
```

Comment:
I thought this one usually has a sidecar?

Reply:
I can remove this one, done in https://github.com/istio/istio/pull/45900/commits/83cbb6ba1459e533588a6a6b6a23356a3213899a